### PR TITLE
Added Exchanges in Nepal

### PR DESCRIPTION
**Pull request by tuladhar**
Added Nepal's first exchange "Bitsewa" and Nepal's flag.

### About Bitsewa

Bitsewa is Nepal's first Bitcoin exchange, which has been operating for nearly 8 months now. Bitsewa exchange provides best rates with automated and instant buying/selling of Bitcoin and many Digital Assets. Bitsewa has been operating under the company called "Bitsewa Pvt. Ltd." and they are legally registered software company in Nepal which can be verified at government company registration website: http://www.ocr.gov.np/index.php/en/

For more details contact me or email at support@bitsewa.com

### CODE REVIEW

**Comment by wbnns** [comment]
@tuladhar Hi there! Thanks for submitting this. Could you please:

1. Change the flag to [this one](https://github.com/gosquared/flags/blob/master/flags/flags-iso/shiny/64/NP.png).
2. Rebase your PR to resolve the conflicts.

Once you've done that this LGTM. 👍

**Comment by tuladhar** [comment]
Hi @wbnns 

Please kindly review the changes.

Hope i didn't messed up anything :)

Thanks

File: _templates/exchanges.html [hunk 1]
```diff
@@ -168,6 +168,12 @@ <h3 id="mexico"><img src="/img/flags/MX.png" alt="Mexican flag">Mexico</h3>
   </div>
   <div>
     <div>
+      <h3 id="nepal"><img src="/img/flags/NP.png" alt="Nepalese flag">Nepal</h3>
+      <p>
+        <a href="https://www.bitsewa.com/">Bitsewa</a><br>
```
Comment (wbnns) [inline _templates/exchanges.html]: @tuladhar Please remove the `<br>` since there is only one entry for Nepal at the moment.

**Comment by tuladhar** [comment]
Thanks @wbnns It has been fixed on commit 7f01f05

**Comment by wbnns** [comment]
@tuladhar Thanks a bunch for working on this. Another PR created a merge conflict here, so I went ahead and took care of adding Bitsewa for you in #1656 so you don't have to do anything else. 👍

**Pull request closed without merging**